### PR TITLE
Changed names on the board page and tested

### PR DESCRIPTION
**Pull request by MerijnJongepier**

### CODE REVIEW

File: src/locales/en/pages.yaml [hunk 1]
```diff
@@ -158,15 +158,15 @@ contact:
...
-    Public relations and chamber music commissioner: Ben Carp ([publicom@krashna.nl](mailto:publicom@krashna.nl) and [kamucom@krashna.nl](mailto:kamucom@krashna.nl))
+    **Board 55 "Klimop"**
+
+    Chairwoman and chamber music commissioner: Ilja de Goede ([voorzitter@krashna.nl](mailto:voorzitter@krashna.nl) and [kamucom@krashna.nl](mailto:kamucom@krashna.nl))\
+    Secretary: Annemarijne Verschuijl ([secretaris@krashna.nl](mailto:secretaris@krashna.nl))\
+    Treasurer: Merijn Jongepier ([penningmeester@krashna.nl](mailto:penningmeester@krashna.nl))\
+    Acquisition commissioner: Mette Klein ([acquisiteur@krashna.nl](mailto:acquisiteur@krashna.nl))\
+    Orchestra commissioner: Jonah Meijer ([orkestcom@krashna.nl](mailto:orkestcom@krashna.nl))\
+    Choir commissioner: Sara Lubbers ([koorcom@krashna.nl](mailto:koorcom@krashna.nl))\
+    Public relations and concert commissioner: Sanne Peters ([publicom@krashna.nl](mailto:publicom@krashna.nl) and [kamucom@krashna.nl](mailto:concertcom@krashna.nl))
```
Comment (MathiasMeuleman) [inline src/locales/en/pages.yaml]: kamucom needs to be updated to concertcom as well

Comment (MerijnJongepier) [reply src/locales/en/pages.yaml]: Oh, thanks, missed that

Comment (MerijnJongepier) [reply src/locales/en/pages.yaml]: Should I make a new pull request?

Comment (MathiasMeuleman) [reply src/locales/en/pages.yaml]: No, it's already merged and deployed

**Pull request merged**